lib/notifications/display.dart

```diff
@@ -281,10 +282,13 @@ class NotificationDisplayManager {
     try {
       // TODO timeout to prevent waiting indefinitely
       final resp = await http.get(url);
-      return resp.bodyBytes;
+      final bodyBytes = resp.bodyBytes;
+      if (resp.statusCode == HttpStatus.ok) {
+        return bodyBytes;
+      }
```

Comment:
nit: Is the variable `bodyBytes` necessary? why not inline `resp.bodyBytes` in the return statement?

```suggestion
      if (resp.statusCode == HttpStatus.ok) {
        return resp.bodyBytes;
      }
```